src/esm_runscripts/filelists.py

```diff
@@ -1276,19 +1276,19 @@ def avoid_overwriting(config, source, target):
         if filecmp.cmp(source, target):
             return target
 
+        warning_function = user_error
         date_stamped_target = f"{target}_{config['general']['run_datestamp']}"
         if os.path.isfile(date_stamped_target):
             if filecmp.cmp(source, date_stamped_target):
                 logger.info(
                     f"File {date_stamped_target} already exists and is identical to "
                     f"the source ({source}). Skipping copying"
                 )
-            elif config["general"]["force_overwrite_in_file_movements"]:
+                return date_stamped_target
+            if config["general"]["force_overwrite_in_file_movements"]:
                 os.remove(date_stamped_target)
+                # Change to user_note, the default was user_error (above)
                 warning_function = user_note
-            else:
-                # This will exit(1) (default in configs/defaults/general.yaml)
-                warning_function = user_error
```

Comment:
I don't see any flaw with the original logic. What is exactly the problem you are trying to resolve here?

Reply:
I revert it most of it as it was, but now we don't get the user_note missing error.